### ISSUE TITLE
In Outlook 2019, NVDA crashes when switching account

### ISSUE THREAD

**Issue by ivnc** [nvaccess/nvda#12559]
Issue sent by a Spanish-speaking user.

### Steps to reproduce:

1. Open Outlook.
2. Press Control+Y to go to another folder.
3. Expand another account anc press Enter on Inbox ("Bandeja de entrada").
[nvda-ofuscated.log](https://github.com/nvaccess/nvda/files/6668614/nvda-ofuscated.log)

### Actual behavior:

NVDA hangs by several minutes and apparently also causes Outlook to hang completely. When running JAWS or Narrator this doesn't happen.

### Expected behavior:

Folder is changed and user can read emails normally.

#### NVDA installed/portable/running from source:

Installed

#### NVDA version:

2020.4

#### Windows version:

Windows 10 21H1 (build 19043.x)

#### Name and version of other software in use when reproducing the issue:

Microsoft Office Outlook 2019

#### Does the issue still occur after restarting your computer?

Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.

2021.1 Beta: same issue.

#### If add-ons are disabled, is your problem still occurring?

Yes

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?

Not applicable / User not asked to

**Comment by ivnc** [comment]
Updated initial comment to precise that the crash only happens when messages are grouped by conversation. Didn't try to reproduce here.

**Comment by Adriani90** [comment]
@ivnc are you still having this issue with NVDA 2024.1 or last alpha?

**Comment by ivnc** [comment]
> @ivnc are you still having this issue with NVDA 2024.1 or last alpha?

Yes, according to the reporter it's still reproducible with 2024.1. thanks.
